label.py

```diff
@@ -1,18 +1,20 @@
-import os
-import json
+import csv
 from PIL import ImageTk, Image
-from Tkinter import Tk, Label
+from tkinter import Tk, Label
 
 import utils
 
 
 try:
-    with open('labels.json', 'r') as f:
-        labels = json.load(f)
+    with open('labels.csv', 'r') as f:
+        reader = csv.reader(f)
+        labels = {row[0]: row[1] for row in reader}
+
 except:
     labels = {}
 
 
+print(labels)
```

Comment:
Let's remove these logging statements, as when there will be a lot of entries in the labels file, this will print a lot of stuff.